src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -353,6 +579,8 @@ public String parseFile(AsciiFile file, BindingResult result) {
                  *
                  * } else {
                  **/
+
+                /*
                 String normalizedFileData = fileParserManager.normalizeDelimiters(filePath, selectedDelimiter, delimiterList, file.getHeaderLineList());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)